sources/packages/web/src/views/institution/locations/programs/LocationPrograms.vue

```diff
@@ -173,22 +138,25 @@ export default defineComponent({
     };
 
     // Pagination sort event callback.
-    const paginationAndSortEvent = async (event: any) => {
-      currentPage.value = event?.page;
-      currentPageLimit.value = event?.rows;
+    const paginationAndSortEvent = async (event: DataTableOptions) => {
+      currentPage.value = event.page - 1;
+      currentPageLimit.value = event.itemsPerPage;
+      const [sortByOptions] = event.sortBy;
       await loadSummary(
-        event.page,
-        event.rows,
-        event.sortField,
-        event.sortOrder,
+        event.page - 1,
```

Comment:
Can you please elaborate on why we cannot use the `event.page` as it is and we must add the `-1` on line 142 and 146?

Reply:
when we were using primevue data table, the value of the page was 0 for the first page and in the vuetify data table the first page value is 1.
eg, this is first page in prime vue 
![image](https://user-images.githubusercontent.com/77353155/221290692-375e0042-077b-40c4-adb1-a360f8d20c19.png)
and in vuetify 3
![image](https://user-images.githubusercontent.com/77353155/221290866-c912297d-a548-4406-a0bb-ea072076a8ff.png)

Reply:
We can change the API to work with the pagination starting in 1 (because 0 based or 1 would be both accaptable) or we can adjust the pagination as needed at the Vue API service layer only, for instance, while calling the method `getPaginationQueryString`.
What I would like to avoid is doing "-1" everywhere.